bin/csv-filter

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'csv'
+
+# Returns an array of converter names.
+# \String +s+ contains a comma-separated list of converter names;
+# each optionally begins with a colon.
+def parse_converters(s)
+  converters = []
+  s.split(',').each do |name|
+    name.sub(/^:/, '')
+    sym = name.to_sym
+    converters.push(sym)
+  end
+  converters
+end
+
+options = {}
+
+OptionParser.new do |p|
+
+  p.program_name = File.basename $0
+  p.version = CSV::VERSION
+  p.release = nil
+  p.summary_indent = ' ' * 4
+  p.banner = <<-EOF
+Usage: #{p.program_name} [options]
+
+  Reads and parses the CSV text content of $stdin' per the given input options.
+  From that content, generates CSV text per the given output options and writes
+  that text to $stdout.
+
+  Three options are similar for input and output:
+
+  - Row separator:
+  
+    - The input row separator is determined:
+  
+      - By option --input_row_sep, if given.
```

Comment:
Could you use `-` not `_` for word separator in option name: `--input-row-sep`

Reply:
Done (for those not commented out -- will get the rest as we go).